docroot/themes/custom/yukonca_glider/templates/paragraphs/paragraph--chart--default.html.twig

```diff
@@ -0,0 +1,96 @@
+ *   - paragraphs: The current template type (also known as a "theming hook").
+ *   - paragraphs--type-[type]: The current paragraphs type. For example, if the paragraph is an
+ *     "Image" it would result in "paragraphs--type--image". Note that the machine
+ *     name will often be in a short form of the human readable label.
+ *   - paragraphs--view-mode--[view_mode]: The View Mode of the paragraph; for example, a
+ *     preview would result in: "paragraphs--view-mode--preview", and
+ *     default: "paragraphs--view-mode--default".
+ * - view_mode: View mode; for example, "preview" or "full".
+ * - logged_in: Flag for authenticated user status. Will be true when the
+ *   current user is a logged-in member.
+ * - is_admin: Flag for admin user status. Will be true when the current user
+ *   is an administrator.
+ *
+ * @see template_preprocess_paragraph()
+ *
+ * @ingroup themeable
+ */
+#}
+
+<div class="chart">
+  <h2>{{ paragraph.field_title.value|raw }}</h2>
+  <figure class="paragraph-items content">
+    <div id="chart-graph-{{ paragraph.id() }}" class="chart-graph"></div>
+    <div class="chart-description">
+      <div aria-hidden="true" class="highcharts-linked-description">
+        <p>
+          <strong class="chart-stat">{{ paragraph.field_highlighted_stat.value|raw }}</strong>
+        </p>
+                          
+        <div class="field field-name-field-chart-description field-type-text-with-summary field-label-hidden">
+          <div class="field-items">
+            <div class="field-item even"> {{ paragraph.field_description.value|raw }} </div>
+          </div>
+        </div>
+      </div>
+      <section class="yukon-department-sections basic-page-collapsable">
+        <div class="panel-group" id="accordion-{{ paragraph.id() }}" role="tablist">
+          <div class="panel panel-default">
+            <div class="panel-heading" role="tab">
+              <h4 class="panel-title">
+                <a class="collapsed" data-toggle="collapse" data-parent="#accordion-{{ paragraph.id() }}" href="#accordion-content-{{ paragraph.id() }}" aria-expanded="false" aria-controls="accordion-content-{{ paragraph.id() }}">
+                  {{ 'Source and notes on data'|t }} <span class="float-right text-blue-400 title-icon"><i class="fa fa-plus-square"></i></span>
+                </a>
+              </h4>
+            </div>
+            <div id="accordion-content-{{ paragraph.id() }}" class="panel-collapse collapse" role="tabpanel">
+              <div class="panel-body">
+                <div class="field field-name-field-source-notes field-type-text-with-summary field-label-hidden">
+                  <div class="field-items">
+                    <div class="field-item even"> {{ paragraph.field_source_notes.value|raw }}</div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+    </div>
+  </figure>
+</div>
+
+<script>
```

Comment:
Please don't mix Scripts and HTML. Create a separate js file and attach it to this template like `{{ attach_library('mytheme/my-library') }}`
https://www.drupal.org/docs/develop/theming-drupal/adding-assets-css-js-to-a-drupal-theme-via-librariesyml#attach-library-specific-twig

Reply:
It was giving issues getting the dynamic IDs and thats why we took another approach of packing the code in our custom module. Please review and let us know if any changes are required.